mocha/utils/apiEndpoints.js

```diff
@@ -3,3 +3,6 @@ exports.feedbackEndpoint = "/api/feedback";
 exports.signUpApiEndpoint = "/api/auth/signup";
 exports.organisationsEndpoint = "/api/organisations";
 exports.postsEndpoint = "/api/posts";
+exports.postApiEndpoint = "/api/posts/{postId}";
+exports.getPostApiEndpoint = "/api/posts/";
```

Comment:
Is this a duplicate? If I'm understanding correctly, I think line 8 "/api/posts" is being used in the test but not this one

Reply:
Corrected